benchmark/utils.sh

```diff
@@ -33,7 +33,7 @@ function get_args() {
     for arg in ${args[@]}
     do
         # set undefined arguments to None
-        if [ -z "$(echo ${!arg})" ]
+        if [ -z "${!arg+x}" ]
```

Comment:
I learned this trick from here: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash